R/seroprevalence_data.R

```diff
@@ -211,8 +284,17 @@ get_sim_probability <- function(sim_data, foi) {
 get_sim_n_seropositive <- function(sim_data,
                                    foi,
```

Comment:
No need to use "Function..." to describe this function. I would instead say, "Simulate conducting a serosurvey"